src/OpenTelemetry.Instrumentation.EntityFrameworkCore/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+* Fixed the `server.address` field in EntityFrameworkCore spans to populate
+  with the server domain name without the `tcp` prefix.
+  ([#2439](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/2439))
```

Comment:
Worth to mention also that `server.port` is also introduced.

Reply:
Good point.

Reply:
done